utils/ticketPacking.js

```diff
@@ -0,0 +1,21 @@
+module.exports = async (obj, type, id, listArr, currentUserId) => {
```

Comment:
```suggestion
module.exports = async ({obj, type, id, listArr, currentUserId}) => {
```
當一個函式有太多參數時，把參數包裝成一個物件會增加可讀性